### PR TITLE
Preview and markdown editor

### PR DESCRIPTION
**Pull request by Mihaiii**
Issues #24 #25 
TODO: update bower.json in case juicy-markdown-tabbededitor package is updated (please see [juicy-markdown-tabbededitor/issues/3](https://github.com/Juicy/juicy-markdown-tabbededitor/issues/3) )

### CODE REVIEW

**Comment by Mihaiii** [comment]
@Mackiovello I will make smaller commits. In this case there are alot of added files in Content/wwwroot/sys . The main changes are in wwwroot\sys\juicy-markdown-tabbededitor\juicy-markdown-tabbededitor.js and \Content\wwwroot\Content\views\CmsItemPage.html

**Comment by Mihaiii** [comment]
@Mackiovello Made smaller commits. Please have another look. :)

**Comment by Mihaiii** [comment]
This would interfere with the wysiwyg editor so doesn't need a review at this moment IMO. Made it before finding out about the other (and more important) issue.

**Comment by Mackiovello** [comment]
Okay. Please tell me when you want the review

**Comment by Mihaiii** [comment]
@Mackiovello Ok, everything seems to be ok now (check passed). Please review when possible. Thanks!

File: src/Content/wwwroot/sys/juicy-markdown-tabbededitor/juicy-markdown-tabbededitor.html [hunk 1]
```diff
@@ -196,15 +196,15 @@
...
+				<juicy-markdown-editor style="height: 400px;" id="mdedit" value="{{value}}"></juicy-markdown-editor>
 			</div>
 			
 			<div>
 				<content-wrapped-juicy-ace-editor value="{{htmlValue::change}}"></content-wrapped-juicy-ace-editor>
 			</div>
 
 			<div>
-				<juicy-markdown value="{{value}}" ghcss="{{ghcss}}"></juicy-markdown>
+				<juicy-markdown style="height: 400px;" value="{{value}}" ghcss="{{ghcss}}"></juicy-markdown>
```
Comment (Mackiovello) [inline src/Content/wwwroot/sys/juicy-markdown-tabbededitor/juicy-markdown-tabbededitor.html]: We don't use inline styling, this is what you should do instead: https://docs.starcounter.io/guides/blending/avoiding-css-conflicts/

**Comment by Mackiovello** [comment]
Please take a look at this one: https://starcounter.gitbooks.io/guidelines/content/contributing-code.html#commit-messages

**Comment by Mackiovello** [comment]
Everything else looks good to me. It's just a little concerning that it puts an `id` attribute on headings that are entered in markdown

**Comment by Mackiovello** [comment]
Please merge

**Pull request merged**